bin/netgrasp

```diff
@@ -45,7 +44,8 @@ if __name__ == "__main__":
 
     # list
     parser_list = subparsers.add_parser("list", help="list devices")
-    parser_list.add_argument("--type", "-t", choices=["device", "event"], default="device", help="type of object to list")
+    parser_list.add_argument("--type", "-t", choices=["device", "event"], default="device",
+                             help="type of object to list")
```

Comment:
Unrelated to functional change -- please split out these non-functional changes.